### PR TITLE
Gallery fixes

### PR DESCRIPTION
**Pull request by icidasset**
Fixes:
https://github.com/ampproject/ampstart/issues/709 
https://github.com/ampproject/ampstart/issues/714
https://github.com/ampproject/ampstart/issues/716
https://github.com/ampproject/ampstart/issues/718

Waiting for an answer on:
https://github.com/ampproject/ampstart/issues/717
https://github.com/ampproject/ampstart/issues/719

Anything else I can take on @bentranter?

### CODE REVIEW

File: css/templates/gallery/components/_sidebar.css [hunk 1]
```diff
@@ -14,9 +14,17 @@
  * limitations under the License.
  */
 
- .gallery-sidebar {
-   width: calc((375 / 16rem));
- }
+:root {
```
Comment (bentranter) [inline css/templates/gallery/components/_sidebar.css]: These need to be in `page-vars.css`, or Google won't be happy

File: css/templates/gallery/components/_sidebar.css [hunk 2]
```diff
@@ -68,8 +70,20 @@
 }
 
 .gallery-sidebar .ampstart-navbar-trigger {
-  padding: calc(10 / 16rem);
-  margin: calc(-10 / 16rem);
+  height: var(--sidebar-nav-trigger-height);
+  line-height: calc(var(--sidebar-nav-trigger-height) * 1.066666667);
+  overflow: hidden;
+  padding: 0 .625rem;
+  margin-top: calc(
```
Comment (bentranter) [inline css/templates/gallery/components/_sidebar.css]: I know this is more readable, but Google also won't like this. It needs to all be on one, giant, hard-to-read line (unfortunately)

File: templates/gallery/partials/tagline.snip.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+<div class="gallery-component-tagline">
+  <div>Gallery of Modern Art</div>
```
Comment (bentranter) [inline templates/gallery/partials/tagline.snip.html]: Google will _probably_ want this to be read from a variable. I know they'll ask use to read from `"heading": "Gallery of Modern&nbsp;Art",`, but the non breaking space will probably make that impossible, so it should be fine as its own new variable in that JSON file.

**Review by bentranter — APPROVED** [review]
Didn't mean to click the review button 😞 don't know how to undo it so I'm submitting it

**Comment by bentranter** [comment]
These fixes should be good, I've got the rest in-progress. Thanks for you help!

File: css/templates/gallery/components/_sidebar.css (hunk 1 above)
Comment (icidasset) [reply css/templates/gallery/components/_sidebar.css]: Sure thing 👍

File: css/templates/gallery/components/_sidebar.css (hunk 2 above)
Comment (icidasset) [reply css/templates/gallery/components/_sidebar.css]: Ok cool. Even though it compiles to a single line?

Comment (bentranter) [reply css/templates/gallery/components/_sidebar.css]: Yup, they're pretty strict about formatting

**Comment by icidasset** [comment]
No problem! I applied your feedback and also fixed the things David just mentioned in Slack.

**Review by bentranter — APPROVED** [review]

**Comment by bentranter** [comment]
You can merge anytime, looks good to me

**Pull request merged**